ANMN/temperature_grid_product/anmn_temp_gridded_product.py

```diff
@@ -229,8 +257,8 @@ def _sample_inter(sample_interval):
             average_window = 60
             n_val_step     = 3
         elif round(sample_interval) >= 1200:
-            average_window = 90
-            n_val_step     = 3
+            average_window = 60
+            n_val_step     = 2
```

Comment:
Why 2?

Reply:
it's the amount of valid measures (good qc) we need in the average window to perform a average. see with @bpasquer